README.md

```diff
@@ -50,7 +50,7 @@ Detailed instructions are provided below.
 ```
 python >= 3.8
 gcc >= 6.4
-protobuf >= 3.18.3
+protobuf >= 3.20.2
```

Comment:
There was a CVE for 3.20.2 so we should move all the way to 3.20.3 instead.

Reply:
Thanks... I can do that!